website/notifications/emails.py

```diff
@@ -147,7 +147,7 @@ def check_node(node, event):
 
 def get_user_subscriptions(user, event):
     user_subscription = NotificationSubscription.load(utils.to_subscription_key(user._id, event))
-    return {key: getattr(user_subscription, key, []) for key in constants.NOTIFICATION_TYPES}
+    return {key: [u._id for u in getattr(user_subscription, key, []).all()] for key in constants.NOTIFICATION_TYPES}
```

Comment:
`{key: getattr(user_subscription, key).all().values_list('guids___id', flat=True) for key in constants.NOTIFICATION_TYPES}`

If you try to call `.all()` on a list it's going to fail and values list will let you get specific fields even across relationships.